src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -672,6 +682,11 @@ function transitionTracks(
         }
     );
 
+    // find the max and min GSVA score value in the next heatmap track group
+    // max and min value is used to create a custom legend for the track group
+    const genesetScoreValueMax = _.max(genesetScoreProfileData) || undefined;
+    const genesetScoreValueMin = _.min(genesetScoreProfileData) || undefined;
```

Comment:
not sure that this does.    if it's 0 we use undefined?

Reply:
No, _max_ and _min_ can return _null_. The interface for _maxProfileValue_ and _minProfileValue_ explains that these props can be a number or undefinedL

```
    maxProfileValue?: number;
    minProfileValue?: number;
```
The ` _.max(genesetScoreProfileData) || undefined` formulation in effect translates _null_ to _undefined_.

But I see that this translation also translated _0_ to _undefined_. This is not the intention. I have reverted the code you mentioned and added the check when the value is actually used:

```
        track.maxProfileValue = genesetScoreValueMax === null? undefined : genesetScoreValueMax;
        track.minProfileValue = genesetScoreValueMin === null? undefined : genesetScoreValueMin;
```